### PR TITLE
Ingested NoDaLiDa, MoTra, and nlp4call

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

**Comment by mjpost** [comment]
If anyone has a moment to review this, NoDaLiDa folks are hoping to have it live in time for the conference tomorrow.

File: data/xml/2021.nodalida.xml [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+      <author><first>Jouni</first><last>Luoma</last></author>
+      <author><first>Li-Hsin</first><last>Chang</last></author>
+      <author><first>Filip</first><last>Ginter</last></author>
+      <author><first>Sampo</first><last>Pyysalo</last></author>
+      <pages>135–144</pages>
+      <abstract>We introduce a corpus with fine-grained named entity annotation for Finnish, following the OntoNotes guidelines to create a resource that is cross-lingually compatible with existing annotations for other languages. We combine and extend two NER corpora recently introduced for Finnish and revise their custom annotation scheme through a combination of automatic and manual processing steps. The resulting corpus consists of nearly 500,000 tokens annotated for over 50,000 mentions categorized into the 18 OntoNotes name and numeric entity types. We evaluate this resource and demonstrate its compatibility with the English OntoNotes annotations by training state-of-the-art mono-, bi- and multilingual deep learning models, finding both that the corpus allows highly accurate recognition of OntoNotes types at 93
+
```
Comment (davidweichiang) [inline data/xml/2021.nodalida.xml:144]: Something went wrong here maybe?

File: data/xml/2021.nodalida.xml [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+      <url hash="9a88f400">2021.nodalida-main.44</url>
+    </paper>
+    <paper id="45">
+      <title>Creating Data in <fixed-case>I</fixed-case>celandic for Text Normalization</title>
+      <author><first>Helga Svala</first><last>Sigurðardóttir</last></author>
+      <author><first>Anna Björk</first><last>Nikulásdóttir</last></author>
+      <author><first>Jón</first><last>Guðnason</last></author>
+      <pages>404–412</pages>
+      <abstract>There is no natural way to acquire normalized data so we try to create good enough data to attempt more advanced methods for text normalization. We manually annotated the first normalized corpus in Icelandic, 40,000 sentences, and developed Regína, a rule-based system for text normalization. Regína gets 90.83
+
```
Comment (davidweichiang) [inline data/xml/2021.nodalida.xml:430]: Same thing again

File: data/xml/2021.nodalida.xml [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+    <paper id="47">
+      <title><fixed-case>D</fixed-case>an<fixed-case>FEVER</fixed-case>: claim verification dataset for <fixed-case>D</fixed-case>anish</title>
+      <author><first>Jeppe</first><last>Nørregaard</last></author>
+      <author><first>Leon</first><last>Derczynski</last></author>
+      <pages>422–428</pages>
+      <abstract>We present a dataset, DanFEVER, intended for multilingual misinformation research. The dataset is in Danish and has the same format as the well-known English FEVER dataset. It can be used for testing methods in multilingual settings, as well as for creating models in production for the Danish language.</abstract>
+      <url hash="b787c42b">2021.nodalida-main.47</url>
+    </paper>
+    <paper id="48">
+      <title>The <fixed-case>I</fixed-case>celandic Word Web: A language technology-focused redesign of a lexicosemantic database</title>
```
Comment (davidweichiang) [inline data/xml/2021.nodalida.xml:465]: Word Web should be fixed-cased

File: data/xml/2021.nodalida.xml [hunk 4]
```diff
@@ -0,0 +1,537 @@
...
+      <title>Getting Hold of Villains and other Rogues</title>
+      <author><first>Manfred</first><last>Klenner</last></author>
+      <author><first>Anne</first><last>Göhring</last></author>
+      <author><first>Sophia</first><last>Conrad</last></author>
+      <pages>435–439</pages>
+      <abstract>In this paper, we introduce the first corpus specifying negative entities within sentences. We discuss indicators for their presence, namely particular verbs, but also the linguistic conditions when their prediction should be suppressed. We further show that a fine-tuned Bert-based baseline model outperforms an over-generating rule-based approach which is not aware of these further restrictions. If a perfect filter were applied, both would be on par.</abstract>
+      <url hash="9bc9ba95">2021.nodalida-main.49</url>
+    </paper>
+    <paper id="50">
+      <title>Talrómur: A large <fixed-case>I</fixed-case>celandic <fixed-case>TTS</fixed-case> corpus</title>
```
Comment (davidweichiang) [inline data/xml/2021.nodalida.xml:486]: Talrómur should be fixed-cased

File: data/xml/2021.nodalida.xml [hunk 5]
```diff
@@ -0,0 +1,537 @@
...
+      <author><first>Amalie Brogaard</first><last>Pauli</last></author>
+      <author><first>Maria</first><last>Barrett</last></author>
+      <author><first>Ophélie</first><last>Lacroix</last></author>
+      <author><first>Rasmus</first><last>Hvingelby</last></author>
+      <pages>460–466</pages>
+      <abstract>We present an open-source toolkit for Danish Natural Language Processing, enabling easy access to Danish NLP’s latest advancements. The toolkit features wrapper-functions for loading models and datasets in a unified way using third-party NLP frameworks. The toolkit is developed to enhance community building, understanding the need from industry and knowledge sharing. As an example of this, we present Angry Tweets: An Annotation Game to create awareness of Danish NLP and create a new sentiment-annotated dataset.</abstract>
+      <url hash="92f983af">2021.nodalida-main.53</url>
+    </paper>
+    <paper id="54">
+      <title><fixed-case>HB</fixed-case> Deid - <fixed-case>HB</fixed-case> De-identification tool demonstrator</title>
```
Comment (davidweichiang) [inline data/xml/2021.nodalida.xml:529]: Deid

**Review by davidweichiang — CHANGES_REQUESTED** [review]

File: data/xml/2021.nodalida.xml (hunk 2 above)
Comment (mjpost) [reply data/xml/2021.nodalida.xml:430]: Hmm, happening at % signs.

File: data/xml/2021.nodalida.xml (hunk 3 above)
Comment (mjpost) [reply data/xml/2021.nodalida.xml]: Fixed this, but didn't add it to the truest.

File: data/xml/2021.nodalida.xml (hunk 4 above)
Comment (mjpost) [reply data/xml/2021.nodalida.xml]: Added to truelist.

**Comment by mjpost** [comment]
@davidweichiang I think I addressed these all.

**Review by davidweichiang — APPROVED** [review]

**Pull request merged**